ui/admin/app/styles/app.scss

```diff
@@ -452,3 +454,103 @@
     }
   }
 }
+
+// secret editor
+.json-secret-editor {
+  .CodeMirror {
+    height: 210px;
+  }
+
+  .CodeMirror-gutters {
+    background: #31332b;
+  }
+}
+
+.secret-editor {
+  background: #272822;
+  border-radius: 4px;
+  height: 210px;
+  margin-bottom: sizing.rems(l);
+}
+
+.skeleton-wrapper {
+  color: var(--ui-gray);
+  display: flex;
+  flex-direction: row;
```

Comment:
This is technically not needed since it's `row` by default but also fine to be explicit

Reply:
Thanks. I thought I cleared up all the unnecessary styling. Removed.